Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -222,6 +236,9 @@ def pull_request_branch_point(cls, repository, args, **kwargs):
                 args.remote = source_remote
         if not source_remote:
             source_remote = repository.default_remote
+        if args.redact and source_remote == repository.default_remote:
+            source_remote = repository.source_remotes()[-1]
```

Comment:
is last remote guaranteed to be security remote?

Reply:
If your checkout is set up correctly, yes.

`source_remotes` are determined by the project (in `metadata/git_config_extension`), not by a user's individual `git` configuration.